packages/core/src/index.ts

```diff
@@ -37,3 +37,4 @@ export { createDefaultResolver } from './module-resolver';
 // low-level api
 export { parseModuleImportStatement, ensureModuleImport } from './helpers/import';
 export { validateCustomPropertyName } from './helpers/css-custom-property';
+export { getStScope } from './features/st-scope';
```

Comment:
why export if we have it on stylable?

Reply:
I prefer not to expose low level APIs at this point, I thought you wanted it as an alternative

Reply:
from what I saw in the code we have stylable instance very close to the usage. we can probably only expose the API

Reply:
removed